INSTALL.md

```diff
@@ -79,10 +79,11 @@ codiumai/pr-agent@v0.9
 git clone https://github.com/Codium-ai/pr-agent.git
 ```
 
-2. Install the requirements in your favorite virtual environment:
+2. Navigate to the `/pr-agent` folder and install the requirements in your favorite virtual environment:
 
+*Tip: make sure Rust is installed and in your `PATH`, instructions: https://rustup.rs*
```

Comment:
why do we need rust ?

Reply:
CLI gave me an error while building the `requirements.txt`, it seems to be a dependency on one of the dependencies, if you want I can find which package it was exactly but will need to uninstall Rust and start a new env

Reply:
After a clean environment it seems to not need it, I don't know what package was causing the error before but I found a few resources online of people mentioning random fixes so I think it may be something external, I will edit it just as a tip if they have the same error